data/mods/Xedra_Evolved/snippets/vampire_speech.json

```
@@ -7,7 +7,10 @@
       "mon_renfield_shotgun",
       "mon_renfield_flamethrower",
       "mon_renfield_bionic_monster",
-      "mon_renfield_abomination"
+      "mon_renfield_abomination",
+      "mon_mi_go",
+      "mon_mi_go_slaver",
+      "mon_mi_go_myrmidon"
```

```suggestion
      "mon_renfield_abomination"
```
Mi-go have more experience with XE world and mindlessly repeat things they've heard less.